web/pgadmin/static/js/Dialogs/index.jsx

```diff
@@ -156,27 +156,33 @@ function masterPassCallbacks(masterpass_callback_queue) {
 export function checkMasterPassword(data, masterpass_callback_queue, cancel_callback) {
   const api = getApiInstance();
   api.post(url_for('browser.set_master_password'), data).then((res)=> {
+    let is_keyring = res.data.data.keyring_name.length > 0;
```

Comment:
Why not simply use `Boolean(res.data.data.keyring_name)`? It will handle null/undefined/empty cases.

Reply:
Using is_keyring as Boolean flag, and null/undefined/empty is handled at backend.